react/SelectBox/SelectBox.jsx

```diff
@@ -200,6 +200,7 @@ const ActionsOption = ({ actions, ...props }) => (
     <span className={withPrefix(props.cx, styles['select-option__actions'])}>
       {actions.map((action, index) => (
         <Icon
+          {...action.props}
```

Comment:
I think it'd better to have `action.iconProps` here, it'd be clearer that the props passed are proxied to the Icon component.